prefect_utils/email_unsubscribes.py

```diff
@@ -0,0 +1,74 @@
+
+import backoff
+import requests
+from prefect import task
+from sailthru.sailthru_client import SailthruClient
+
+from . import snowflake
+
+
+@task
+@backoff.on_exception(backoff.expo,
+                      snowflake.connector.ProgrammingError,
```

Comment:
Is there also a requests exception/HTTP status code ("too many calls") that should be handled here from the Braze call? If so, should those Braze API calls be broken out into a separate method with a separate backoff - to avoid re-starting the method by re-querying Snowflake?

Reply:
Yeah, good call, I'll look at that.

Reply:
Fixed.